content/php-client/resources/attribute-groups.md

```diff
@@ -0,0 +1,48 @@
+## Attribute group
+
+### Get an attribute group
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+/*
+ * Returns an array like this:
+ * [
+ *     'code'       => 'info',
```

Comment:
This example is weird. Better use "marketing" or "technical" as an attribute groupe name. Do not hesitate to use the same example than the one you used in this PR: https://github.com/akeneo/pim-api-docs/commit/52115012032f4ee48320b7fa77eaa817bc685bbb